Dockerfile

```diff
@@ -4,8 +4,6 @@ COPY ./frontend /frontend
 
 WORKDIR /frontend
 
-RUN rm -rf node_modules
```

Comment:
Any reason for removing this line?

Reply:
Since it's been added to the `.dockerignore` it shouldn't be copied into the image. So I figured remove a redundant layer. I can add this back in for safety if we want.